src/odemis/gui/comp/combo.py

```diff
@@ -189,15 +189,18 @@ class ColorMapComboBox(ComboBox):
 
     def __init__(self, *args, **kwargs):
         """
-        colormap_dict: dict of str -> matplotlib colormap. Corresponds to a
-        name of the color map to a color map object
+        labels: list of str that are the names of color maps
+        choices: list of matplotlib colormap. List of corresponding color map objects
         """
-        self.colormap_dict = kwargs['colormap_dict']
-        self.choices = self.colormap_dict.keys()
-        kwargs['choices'] = self.choices
+        labels = kwargs['labels']
+        colormaps = kwargs['choices']
+        self.choices = labels
+        kwargs['choices'] = labels
```

Comment:
I don't think that makes sense. The super normally set as client data the "choices". So if you were not changing "choices", it should work out-of-the-box, and you wouldn't need to call `.SetClientData()` here.